### DISCUSSION TITLE
What are the params for listPlugin?

### DISCUSSION THREAD

**Discussion by MetalCar** [mdx-editor/editor#512] [Q&A]
In the documentation, I found the function listsPlugin. There is a parameter documented, but I am unsure what to enter for it.

What I want to do is disable ordered lists. We have a problem with rendering a date like "18. January 2024". The list automatically renders it as "1. January 2024". Can someone help?

**Comment by petyosi** [comment]
I don't think that there's a way to disable this; This is how https://github.com/syntax-tree/mdast-util-from-markdown parses it.
